### ISSUE TITLE
[Feature] Custom path setting for wildcard

### ISSUE THREAD

**Issue by ujl123** [runew0lf/RuinedFooocus#197]
Taking a look at the path manager, it doesn't look like there's a` "path_wildcards"`.  could this be added as a feature so wildcards can also be added in paths.json?

**Comment by DavidDragonsage** [comment]
Yes, I second that request. RuinedFooocus is the only UI platform that does not allow me to share wildcards.

**Comment by yownas** [comment]
Jeez.. Don't know why I missed to fix this one. Oh, well, better late than never. b9520bdf51f07ac69cff5b249aae02869f1a11ac

**Comment by DavidDragonsage** [comment]
@yownas Thank you. All the best!
